### PR TITLE
Chore/docs console warnings

### PR DESCRIPTION
**Pull request by saltcod**
Fix warnings and errors with missing/dupe keys

### CODE REVIEW

File: apps/docs/components/Navigation/NavigationMenu/NavigationMenu.tsx [hunk 1]
```diff
@@ -160,17 +160,17 @@ const SideNav = () => {
...
-      <NavigationMenuGuideList id={'auth'} />
-      <NavigationMenuGuideList id={'functions'} />
-      <NavigationMenuGuideList id={'realtime'} />
-      <NavigationMenuGuideList id={'storage'} />
-      <NavigationMenuGuideList id={'platform'} />
-      <NavigationMenuGuideList id={'resources'} />
-      <NavigationMenuGuideList id={'integrations'} />
-      <NavigationMenuGuideList id={'self_hosting'} />
-      <NavigationMenuGuideList id={'reference'} />
+      <NavigationMenuGuideList id={'gettingstarted'} context={context} />
```
Comment (saltcod) [inline apps/docs/components/Navigation/NavigationMenu/NavigationMenu.tsx:163]: We need to pass something in here ("side" or "mobile" as we're rendering all these menus twice. 

We need a way to differentiate the keys on the mapped items.

File: apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts [hunk 2]
```diff
@@ -688,22 +688,22 @@ export const reference = {
...
-        {
-          name: 'supbase-python',
-          url: '/reference/python/start',
-          level: 'reference_python',
-          items: [],
-          icon: 'docs/img/icons/javascript.svg',
+          icon: '/img/icons/menu/reference-dart',
         },
+        // {
+        //   name: 'supbase-python',
```
Comment (dannykng) [inline apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts:701]: 😄 
```suggestion
        //   name: 'supabase-python',
```

File: apps/docs/components/Navigation/NavigationMenu/NavigationMenu.tsx (hunk 1 above)
Comment (MildTomato) [reply apps/docs/components/Navigation/NavigationMenu/NavigationMenu.tsx:163]: fixed in another PR

Comment (MildTomato) [reply apps/docs/components/Navigation/NavigationMenu/NavigationMenu.tsx:163]: https://github.com/supabase/supabase/pull/10930

just fixing mobile stuff now.
fixed the duplicate menu issue locally, about to push

**Review by MildTomato — APPROVED** [review]

**Pull request merged**